src/corner_kick/src/containers/mainUI/GlobalStyle.tsx

```diff
@@ -0,0 +1,186 @@
+/**
+ * This file defines the main styling of the application and connects the
+ * theme to all styled-components instances
```

Comment:
Separated layout from theming. Layout is now defined in [style.css](https://github.com/qgolsteyn/Software/blob/master/src/corner_kick/src/style.css) and theming is defined in [Theme.tsx](https://github.com/qgolsteyn/Software/blob/master/src/corner_kick/src/containers/mainUI/Theme.tsx).

Reply:
I don't see `style.css` in this PR? Looks like that link goes to `master` on your fork, but this PR is from `destruction`?

Reply:
Thoughts on changing `style.css` to `grid_layout.css` (or similar)? I think it would make more sense if one were trying to hunt down where the layout was.

Reply:
Moved layout to `index.html`